test/prototype/test_with_asset.py

```diff
@@ -135,6 +137,8 @@ def test_vocab_from_file(self) -> None:
         self.assertEqual(v.get_itos(), expected_itos)
         self.assertEqual(dict(v.get_stoi()), expected_stoi)
 
+    # TODO(Nayef211): remove decorator once https://github.com/pytorch/text/issues/1900 is closed
+    @unittest.skipIf(platform.system() == "Linux", "Test is known to fail on Linux.")
```

Comment:
Every Linux? or CI? If it's only in CI, perhaps this should be restricted to CI with `"CI" in os.environ`

Reply:
Let me add that as a condition as well